resources/views/backend/categories/index.blade.php

```diff
@@ -0,0 +1,128 @@
+                                                @endif
+                                                <td class="text-right">
+                                                    <a href="{{ url('admin/categories/'.$cate->id.'/edit') }}" id="update" class="btn btn-circle btn-outline btn-primary" ><i class="fa fa-edit"></i></a>
+                                                    <a data-toggle="modal" data-target="#confirmDelete" id="del" class="btn btn-circle btn-outline btn-danger btnDel"><i class="fa fa-trash-o"></i></a>
+                                                    <input type="hidden" value="{{ $cate['id'] }}">
+                                                </td>
+                                            </tr>
+                                        @endforeach
+                                    </tbody>
+                                </table>
+                            </div>
+                            <!-- /.table-responsive -->
+                            {!! $cates->links() !!}
+                        </div> 
+                        <!-- /.panel-body -->
+                    @else
+                        <div class="alert alert-info">
+                            <strong>Info!</strong> There are no brands.
+                        </div>
+                    @endif
+                </div>
+                <!-- /.panel -->
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+        <!-- /.row -->
+    </div>
+
+    @if (count($cates) > 0)
+    <div class="modal fade" id="confirmDelete" role="dialog" aria-labelledby="confirmDeleteLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title">Delete a category "<span id='idDel'></span>"</h4>
+                </div>
+                <div class="modal-body text-center alert alert-danger">
+                    <h3 class="text-danger">Are you sure delete this category?</h3>
+                </div>
+                <div class="modal-footer">
+                    <form action="{{ url('admin/categories/'.$cate->id) }}" method="POST">
+                        {!! csrf_field() !!}
+                        {!! method_field('DELETE') !!}
+                        <button type="submit" class="btn btn-danger btn-sm">Delete</button>
+                        <button type="button" class="btn btn-primary btn-sm pull-right" data-dismiss="modal">Cancel</button>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </div>
+    @endif
+
+    <script>
+        $(document).ready(function(){
+            $(document).on('click',".btnDel", function(){
```

Comment:
[rank B] don't use this as less as good. You should write like:
`$(".btnDel").on('click', function(){ ....`
And if you can define id for html element, please use it when using selector.